SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DormancyTests/DormancyTest.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "DormancyTest.h"
+
+#include "DormancyTestActor.h"
+
+AActor* ADormancyTest::CreateDormancyTestActor()
+{
+	return GetWorld()->SpawnActor<ADormancyTestActor>({ 0.0f, 0.0f, 0.0f }, FRotator::ZeroRotator);
+}
+
+void ADormancyTest::CheckDormancyAndRepProperty(const TEnumAsByte<enum ENetDormancy> ExpectedNetDormancy, const int ExpectedTestIntProp)
```

Comment:
nit - but perhaps rename Check to Require to indicate what it's doing internally